packages/backend/src/plugins/catalog.ts

```diff
@@ -50,7 +50,7 @@ export default async function createPlugin(
     ...env,
     groupNamingStrategy: 'kebab-case-name',
     userNamingStrategy: 'strip-domain-email',
-  });
+  })[0];
```

Comment:
why only add the first one?

Reply:
I am not aware of the account being configured, the [app-config](https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/app-config.yaml) does not have any value for `catalog.providers.okta`